amy/workshops/tests/test_template_tags.py

```diff
@@ -0,0 +1,47 @@
+from django.test import TestCase
+from flags.sources import Condition, Flag
+
+from workshops.templatetags.feature_flag_conditions import (
+    first_parameter_condition,
+    url_strip_param_value,
+)
+
+
+class TestFeatureFlagConditions(TestCase):
+    def test_first_parameter_condition(self):
+        # Arrange
+        condition1 = Condition(condition="session", value="test")
+        condition2 = Condition(condition="parameter", value="test")
+        condition3 = Condition(condition="boolean", value="test")
+        flag = Flag(name="TEST_FLAG", conditions=[condition1, condition2, condition3])
+        # Act
+        result = first_parameter_condition(flag)
+        # Assert
+        self.assertEqual(result, condition2)
+
+    def test_first_parameter_condition__not_found(self):
+        # Arrange
+        condition1 = Condition(condition="session", value="test")
+        condition2 = Condition(condition="user", value="test")
+        condition3 = Condition(condition="boolean", value="test")
+        flag = Flag(name="TEST_FLAG", conditions=[condition1, condition2, condition3])
+        # Act
+        result = first_parameter_condition(flag)
+        # Assert
+        self.assertIsNone(result)
+
+    def test_url_strip_param_value(self):
```

Comment:
Is there a case where a full url will be passed to this function? If so, should it also handle the case where multiple params are passed?
If it's just for stripping values from parameter conditions, the name could be changed to make that clearer

Reply:
Would it sound better as `url_strip_query_string`?

Reply:
Realistically this function should not be used to strip URLs out of the parameter value. I'll change the name to `parameter_strip_value` - the `parameter` sounds generic but in the context of specific [django flag condition](https://cfpb.github.io/django-flags/conditions/#parameter) it should be clearer.